crates/resolver/src/lib.rs

```diff
@@ -249,12 +249,6 @@ pub use async_resolver::AsyncResolver;
 pub use async_resolver::TokioAsyncResolver;
 pub use hosts::Hosts;
 
-/// This is an alias for [`AsyncResolver`], which replaced the type previously
-/// called `ResolverFuture`.
-#[deprecated(note = "use [`hickory_resolver::AsyncResolver`] instead")]
-#[cfg(feature = "tokio-runtime")]
-pub type ResolverFuture = TokioAsyncResolver;
```

Comment:
maybe we need a breaking change comment about this in the changelog.

Reply:
Personally I think that would be overkill: the deprecation has been around since #487, 6 years ago, and it has been a long time since that name even made sense in an idiomatic understanding of async Rust.